tests/format/less/case/case.less

```diff
@@ -145,3 +145,26 @@ a:AFTER {
 TABLE {}
 
 .foo { &-KeepSelector {} &-KeepSelector & .KeepClassSelector {} &-100\.200 {} }
+
+@number: 49;
+@keyframes loading {
+    .circle(@index) when (@index < @number){
+        @i: @index+1;
+        @x: round((@i / @number) * 100,2);
+        @selector: e('@{x}%');
+        @{selector}{
+            background-image: url('../../assets/image/loading/1_@{i}.png');
+        } 
+        .circle(@i)
+    }
+    .circle(0);
+}
+
+.loading {
+    width: 100px;
+    height: 100px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    animation: loading 1s infinite;
+}
```

Comment:
Thanks for add this test, would you mind put the code into a separate file?

There are already many lines in this file. Maybe a file named `variable-selector.less` or something?

Reply:
Maybe we should also just test the minimal reproduction code  `@{selector}{}`

Reply:
Thank you very much. I adjusted it again